.github/workflows/release.yml

```diff
@@ -23,6 +23,7 @@ jobs:
         with:
           token: ${{ steps.app-token.outputs.token }}
       - run: npm install --no-save @semantic-release/git semantic-release-plugin-github-breaking-version-tag
-      - run: npx semantic-release
+      - run: npm run build
+      - run: npx semantic-release --debug
```

Comment:
just in case it doesn't work, and for future problems with the release automation, the `--debug` flag will give us more logs for debugging